steam_wrapper/steam_wrapper.py

```diff
@@ -412,6 +429,19 @@ def enable_discord_rpc():
             os.symlink(src=src_rel, dst=dst)
 
 
+def check_extensions(flatpak_info):
+    _, _, _, runtime_branch = flatpak_info["runtime"].split("/")
+    installed_ext_ids = set(flatpak_info["runtime-extensions"]) | \
+                        set(flatpak_info["app-extensions"])
+    for ext_id in ["org.freedesktop.Platform.Compat.i386",
+                   "org.freedesktop.Platform.GL32.default"]:
+        if ext_id not in installed_ext_ids:
```

Comment:
I would prefer these two be combined into same error.

Reply:
The implementation is going to be rather ugly without a real template engine, but well, done.